### DISCUSSION TITLE
Deprecation Warning

### DISCUSSION THREAD

**Discussion by magistrator** [inocan-group/vue3-google-map#292] [Q&A] [answered]
I'm getting this in console
`
As of February 21st, 2024, google.maps.Marker is deprecated. Please use google.maps.marker.AdvancedMarkerElement instead. At this time, google.maps.Marker is not scheduled to be discontinued, but google.maps.marker.AdvancedMarkerElement is recommended over google.maps.Marker. While google.maps.Marker will continue to receive bug fixes for any major regressions, existing bugs in google.maps.Marker will not be addressed. At least 12 months notice will be given before support is discontinued. Please see https://developers.google.com/maps/deprecations for additional details and https://developers.google.com/maps/documentation/javascript/advanced-markers/migration for the migration guide.`

I think the code need to be updated

**Comment by Shujee** [comment]
I too am getting this. Any ETA on when this will be updated?

**Comment by JoseGoncalves** [comment, marked as answer]
To avoit that warning you need to switch to the [AdvancedMarker](https://github.com/inocan-group/vue3-google-map?tab=readme-ov-file#advanced-marker) component.

**Comment by cirolosapio** [comment]
I also get the warning using the [CustomMarker](https://vue3-google-map.com/components/custom-marker.html)
is possible to use a custom marker with AdvancedMarker or the CustomMarker should be updated?
